### ISSUE TITLE
Better contrast on action buttons

### ISSUE THREAD

**Issue by Elchi3** [opencollective/opencollective#3943]
**Describe the bug**
3.5:1 contrast on action button text isn’t good enough. 

**To Reproduce**
Feedback is from the W3C slack re the OWD page (https://opencollective.com/open-web-docs)

**Expected behavior**
Enough contrast to not exclude disabled devs

**Additional context**
CC'ing @sideshowbarker who brought this feedback to me and who might be able to provide the context that I'm missing here to further describe the issue.

**Comment by AutoSponge** [comment]
I have trouble reading many of the interactive elements of the page. I confirmed my suspicions using [axe](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd), but many tools can help you find these issues. Contrast ratios should be between 4.5:1 ([wcag AA](https://www.w3.org/TR/WCAG21/#contrast-minimum)) and 15:1 (higher causes other issues like triggering my astigmatism). More important elements should probably be more visible, not less, so 7:1 is a good benchmark (also the [wcag AAA](https://www.w3.org/TR/WCAG21/#contrast-enhanced) requirement).

**Comment by Betree** [comment]
Thank you both for documenting this. Open Collective design team has an open project to improve the contrasts of texts on the website globally. In the meantime, I'm going to push a small PR to improve things where it's simple.

**Comment by Betree** [comment]
Most of the issues for the profile page are now fixed. The main one still standing is about the action buttons, which have a `4.19:1` ratio - still below the `4.5:1` minimum.

![image](https://user-images.githubusercontent.com/1556356/107378109-f1b59480-6aeb-11eb-8521-679190c15ef3.png)

**Comment by Betree** [comment]
This will be resolved by https://github.com/opencollective/opencollective/issues/4045
